### DISCUSSION TITLE
How to disable automatically kubectl

### DISCUSSION THREAD

**Discussion by ehvs** [JanDeDobbeleer/oh-my-posh#5796] [Troubleshoot]
Hi, is there a way to only have kubectl toggled on if I ran a kubectl command? Otherwise it keeps on all the time.
- Ive tried already `cache_timeout` to `1` but no luck.

**Comment by JanDeDobbeleer** [comment]
Depending on the shell you can use [tooltips](https://ohmyposh.dev/docs/configuration/tooltips) for this.
